webpack/assets/javascripts/react_app/components/common/DateTimePicker/DateTimePicker.js

```diff
@@ -78,13 +78,13 @@ class DateTimePicker extends React.Component {
       <div>
         <InputGroup className="input-group date-time-picker-pf">
           <FormControl
-            {...inputProps}
             aria-label="date-picker-input"
             type="text"
             className="date-time-input"
             name={name}
             value={hiddenValue && !required ? '' : formatDateTime(value)}
             onChange={e => this.setSelected(e.target.value)}
+            {...inputProps}
```

Comment:
I think instead of overriding it here it should be better to send the correct name here:
https://github.com/theforeman/foreman/blob/b10e4c081eafe28e170b2b3de1be83a069c7fd08/webpack/assets/javascripts/react_app/components/Template/TemplateInput.js#L49
and in the form props here:
https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/forms/DateTime/DateTime.js

Reply:
I was thinking about it originally, but this way it was a smaller change :) Changed